README.md

```diff
@@ -12,22 +12,35 @@ Piwik is the leading open-source analytics platform that gives you more than jus
 
 ![logo](https://rawgit.com/piwik/docker-piwik/master/logo.svg)
 
-# How to use this image
+## Usage
+
+In keeping with a 'pure' micro-services approach, this image runs a Piwik service only (in the form of FastCGI). Because of that it **must** be used with companion containers which provide a database for data storage and HTTP to FastCGI proxy/translation services for the user interface.
+
+## Pulling
+
+You can use this command to pull the image manually before runtime:
+
+```bash
+sudo docker pull piwik:latest
+```
+
+## Runtime
+
+You can run the Piwik container and service like so:
 
 ```bash
-docker run --link some-mysql:db -d piwik
+docker run -d --link some-mysql:db piwik:latest
```

Comment:
actually, looking at other official images, we can remove the latest and the -d all together for clarity purpose.